### DISCUSSION TITLE
any tip to show file count / distribution size?

### DISCUSSION THREAD

**Discussion by masterkain** [verdaccio/verdaccio#2381] [Q&A]
I guess I have to integrate some tools on my CI? any name?

![Screenshot 2021-08-18 at 00 01 13](https://user-images.githubusercontent.com/12844/129806523-f8b88939-2947-4ac7-befb-1d6f8b7cb249.png)

**Comment by juanpicado** [comment]
No idea, actually the **file size** might be easy to achieve with verdaccio eventually (`fs.stats`) after publish a file, but the file count will require unpack, count etc .... that's much harder.

Anyway, most of those stats comes from the `uplinks`, in other words `npmjs`, because they do have capacity to achieve it.
